app/src/main/res/layout/activity_splash_screen.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/splash_background"
+    android:theme="@style/splashScreenTheme"
+    tools:context=".SplashScreenActivity">
+
+    <ImageView
+        android:id="@+id/splash_screen_title"
+        android:layout_width="200dp"
+        android:layout_height="200dp"
+        android:layout_marginStart="30dp"
```

Comment:
I'd suggest to extract all these values to dimens.xml in order to get used to do it this way.
Also, based on the discussion about standard sizes, let's use 32dp.